blocks/tabs/tabs.css

```diff
@@ -0,0 +1,72 @@
+.tabs .tabs-list {
+  display: flex;
+  gap: 8px;
+  max-width: 100%;
+  overflow-x: auto;
+  flex-direction: row-reverse;
+}
+
+@media screen and (max-width:360px) { 
```

Comment:
You should use mobile-first approach, render mobile view as default and override for desktop view

Reply:
updated